Async SGLang rollout sharding manager missing offload_param config

as the title, using sglang_async with offload_param: True, will cause an error on fsdp sharding manager like:

```
  File "/nfs/volume-1615-1/lilinchuan/project/verl2/verl/single_controller/ray/base.py", line 418, in func
    return getattr(self.worker_dict[key], name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nfs/volume-1615-1/lilinchuan/project/verl2/verl/single_controller/base/decorator.py", line 420, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/nfs/volume-1615-1/lilinchuan/project/verl2/verl/workers/fsdp_workers.py", line 574, in generate_sequences
    with self.rollout_sharding_manager:
  File "/nfs/volume-1615-1/lilinchuan/project/verl2/verl/workers/sharding_manager/fsdp_sglang.py", line 109, in __enter__
    params = self.module.state_dict()
             ^^^^^^^^^^^^^^^^^^^^^^^^
......
  File "/home/luban/anaconda3/lib/python3.11/site-packages/torch/distributed/fsdp/_runtime_utils.py", line 290, in _unshard
    ran_pre_unshard = handle.pre_unshard()
                      ^^^^^^^^^^^^^^^^^^^^
  File "/home/luban/anaconda3/lib/python3.11/site-packages/torch/distributed/fsdp/_flat_param.py", line 1270, in pre_unshard
    self._check_on_compute_device(self.flat_param)
  File "/home/luban/anaconda3/lib/python3.11/site-packages/torch/distributed/fsdp/_flat_param.py", line 2537, in _check_on_compute_device
    _p_assert(
  File "/home/luban/anaconda3/lib/python3.11/site-packages/torch/distributed/utils.py", line 168, in _p_assert
    raise AssertionError(s)
AssertionError: Expects tensor to be on the compute device cuda:0, was on cpu
```

--- a/verl/workers/fsdp_workers.py
+++ b/verl/workers/fsdp_workers.py
@@ -457,6 +457,7 @@ class ActorRolloutRefWorker(Worker):
                 model_config=self.actor_model_config,
                 full_params="hf" in self.config.rollout.load_format,
                 device_mesh=rollout_device_mesh,
+                offload_param=self._is_offload_param,
             )
             log_gpu_memory_usage("After building sharding manager", logger=None)
 

